final_prep/README.md

```diff
@@ -32,24 +33,29 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+Week 1
+- ![](https://user-images.githubusercontent.com/88012780/132966404-d8e2dbeb-ab64-4d81-9eda-045120369b18.png)
+
+Week 2
+- ![](https://user-images.githubusercontent.com/88012780/132966410-1b7cef1e-89d4-422c-bd39-cda39b22568e.png)
+
+Week 3
+- ![](https://user-images.githubusercontent.com/88012780/132966426-01512be9-f202-444d-b4d2-82fc9c9932cf.png)
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] [Link to gist](https://gist.github.com/arnaldoaparicio/382fa5eb4743e5bda07a770e47dcfbe5)
```

Comment:
For your introduction message to your mentor, do not bullet point. You should write in prose (paragraph form) and treat it as a professional relationship. Let me know if you'd like me to re-review (via slack dm).